app/javascript/components/lease_ups/LeaseUpsStatusModalWrapper.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+import Dropdown from '../molecules/Dropdown'
+import FormModal from 'components/organisms/FormModal'
+import { TextArea } from 'react-form'
+import formUtils from '~/utils/formUtils'
+import { LEASE_UP_STATUS_OPTIONS, getLeaseUpStatusStyle } from './leaseUpsHelpers'
+
+class LeaseUpsStatusModalWrapper extends React.Component {
+  formValidator = (values) => {
+    return {
+      comment: !values.comment || values.comment.trim() === '' ? 'Please provide a comment.' : null,
+    };
+  };
+
+  render() {
+    return (
+      <FormModal
+        header="Update Status"
+        primary="update"
+        secondary="cancel"
+        isOpen={this.props.isOpen}
```

Comment:
about `props` I would follow this conventions: Instead of referencing `this.props.whatever` I would extract the props in the first line in the render method like `const { whatever } = this.props` so you can see what properties are being using in the render method and you do not have to scan the code to see what it's being used. Also, we save a few characters in the JSX, but this is minor improvement :P